### ISSUE TITLE
Live Server doesn't open Firefox and Chrome (Browser Crash)

### ISSUE THREAD

**Issue by jaawaad** [ritwickdey/vscode-live-server#622]
## I'm submitting a...

```html
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Feature request
[ ] Documentation issue or request
[ ] Other: <!-- Please describe: -->
```

## Current behavior

After click on "Go Live" on Vscode, **Firefox** showed crash popup.
On **Google Chrome** a couple minutes after opened, the browser crashed and closed without any message.
This problem also there on PrivateMode.

**Error screenshot on Firefox:**
![Screenshot_2019-12-10 Screenshot from 2019-12-10 13-54-17 png (PNG Image, 3286 × 1080 pixels)](https://user-images.githubusercontent.com/57540859/70524201-ea969d80-1b59-11ea-800c-89426f29530b.png)
## Environment

```html
Browser:

- [x] Chrome (desktop) version  78.0.3904.108 
- [x] Firefox version 70.0.1 

For Tooling issues:

- Live Server: 5.6.1
- Platform: Linux (Ubuntu 19.10)
- Visual Studio Code: 1.40.2
```

**Comment by stancualbertstefan** [comment]
Hy,I do have the same problem. 
But for Chrome works. (I have to specify that Chrome is installed by downloading the 64x.deb packadge from https://www.google.com/chrome/ and installed via GUI on ubuntu 20.04)  

The problem is Firefox that keeps crashing on every iteration trying to start live server from VsCode.

Hope someone can have a solution to this.

**Comment by layne74** [comment]
I get the same issue, but whats interesting is that if i open a link within VSC ( such as if you click on the Settings Docs from the extension page ) it opens a new Firefox window, then when I start live server it opens perfectly in a new tab in that new Firefox window.

**Comment by Lan-ces** [comment]
I see that in the source code gives this error:

console.error('Upgrade your browser. This Browser is NOT supported WebSocket for Live-Reloading.');

**Comment by stancualbertstefan** [comment]
for me,was the latest vers of Firefox on ubuntu 20.04

**Comment by Mario-td** [comment]
I opened the localhost with the browser on the port number that vscode specified and it worked for me

**Comment by hendro2019** [comment]
I am having this same issue using ubuntu 20.04. I tried switching the browser to chrome, and nothing happens when I try to launch live server. I also tried typing the url directly "localhost/path-to-file/" and that didn't work either. Still looking for a good work around

**Comment by Mario-td** [comment]
did you try with "localhost:vscode-port-number"?

**Comment by hendro2019** [comment]
Yes I did. The bug has to do with using a snap installation instead of deb. I was able to resolve the issues using ctaggart's workaround (#75579)

**Comment by Mystic-Lightning** [comment]
I am having the same issue i am using ubuntu 21.04 and firefox 89.0.2 it shows mozilla crashed if forefox isnt already running
but works fine if firefox is opened before i start the live server
Error screenshot:
![Screenshot from 2021-07-07 19-43-47](https://user-images.githubusercontent.com/73812009/124774883-c5e3f100-df5b-11eb-91e4-eee515d1b6a1.png)

**Comment by Jrwolf99** [comment]
I followed ctaggart's workaround (https://github.com/microsoft/vscode/issues/75579) and was able to resolve this issue.

1. Remove and purge the current installation `sudo apt remove code --purge`
2. Download and install the deb package from https://code.visualstudio.com/download
`sudo apt install ./<vscodefile>.deb`

_Originally posted by @AsianJohnBoi in https://github.com/microsoft/vscode/issues/85344#issuecomment-574963645_

**Comment by stifler97** [comment]
@Jrwolf99 Thanks. I followed your solution and it came out to work for me. I had installed `vscode` from snap and that caused all the issues.
